src/main/scala/net/ripe/rpki/publicationserver/AppConfig.scala

```diff
@@ -72,6 +72,11 @@ class AppConfig {
       case Success(data) => data
     }
 
+  lazy val repositoryFlushInterval = getConfig.getDuration(
```

Comment:
It would be nice to also remove the `snapshotSyncDelay`, this name makes more sense, but they are essentially the same values.